Fix TE microbenchmark harness to use appropriate fuser/executor

bertmaher opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #42901 Add a microbenchmark for LSTM elementwise portion
* **#42900 Fix TE microbenchmark harness to use appropriate fuser/executor**
* #42899 Speed up CUDA kernel launch when block/thread extents are statically known

Differential Revision: [D23079715](https://our.internmc.facebook.com/intern/diff/D23079715)

bertmaher committed:
Fix TE microbenchmark harness to use appropriate fuser/executor

[ghstack-poisoned]

ZolotukhinM commented on benchmarks/tensorexpr/__main__.py:
```diff
@@ -92,7 +92,15 @@ def main():
+        import torch
+        torch._C._jit_set_profiling_executor(False)
+        torch._C._jit_set_texpr_fuser_enabled(False)
+        torch._C._jit_set_profiling_mode(False)
```
Nit: technically `set_profiling_mode` is unneeded after we call `_jit_set_profiling_executor(False)`. Also, since the fuser was specifically requested to be "old" it might make sense to turn it on explicitly rather than relying on a default value (` torch._C._jit_override_can_fuse_on_gpu(True)`)

ZolotukhinM reviewed (approved):
Looks good!

bertmaher committed:
Update on "Fix TE microbenchmark harness to use appropriate fuser/executor"


Differential Revision: [D23079715](https://our.internmc.facebook.com/intern/diff/D23079715)

[ghstack-poisoned]

bertmaher committed:
Update on "Fix TE microbenchmark harness to use appropriate fuser/executor"


Differential Revision: [D23079715](https://our.internmc.facebook.com/intern/diff/D23079715)

[ghstack-poisoned]
